Dockerfile

```diff
@@ -1,6 +1,16 @@
 FROM olantwin/ship-base:20180627-online
 
-ADD . /FairShip
+# Copy FairShip scripts
+COPY . /FairShip
 
-RUN aliBuild -c shipdist/ --defaults fairship build FairShip --no-local ROOT \
-	&& aliBuild clean --aggressive-cleanup
```

Comment:
Why remove the cleanup? I prefer keeping the image size as small as possible.

Reply:
Agree. I'll put it back. I removed it for tests, but now we can put it back.

Reply:
Thanks!